google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpc.java

```diff
@@ -287,8 +287,6 @@ StreamingCall read(
   ApiFuture<ResultSet> executeQueryAsync(
       ExecuteSqlRequest request, @Nullable Map<Option, ?> options);
 
-  ResultSet executePartitionedDml(ExecuteSqlRequest request, @Nullable Map<Option, ?> options);
```

Comment:
Why do we remove this? This seems to be related to the failed test: 

```bash
[ERROR] 7002: com.google.cloud.spanner.spi.v1.GapicSpannerRpc: Method 'public com.google.spanner.v1.ResultSet executePartitionedDml(com.google.spanner.v1.ExecuteSqlRequest, java.util.Map)' has been removed
[ERROR] 7002: com.google.cloud.spanner.spi.v1.SpannerRpc: Method 'public com.google.spanner.v1.ResultSet executePartitionedDml(com.google.spanner.v1.ExecuteSqlRequest, java.util.Map)' has been removed
```

Reply:
This method is no longer used (instead the client library uses the streaming version), but the removal of it should be moved to a separate PR. I've reverted this change. That should also fix the clirr build failure.